### PR TITLE
Top down edits including website

### LINKED ISSUE #154: 3.2.5 Validation of authority
The policy requirements in Section 3.2.5 can be confusing or misconstrued in the scope of this CP.  For example, the use of "organization". 

Appreciate any / all suggestions for updates that add clarity. 

> ### 3.2.5 Validation of authority
> If the Applicant for a Certificate containing Subject Identity Information is an organization, the CA SHALL use a Reliable Method of Communication to verify the authenticity of the Applicant Representative's certificate request.
> 
> The CA MAY use the sources listed in section 3.2.2.1 to verify the Reliable Method of Communication. Provided that the CA uses a Reliable Method of Communication, the CA MAY establish the authenticity of the certificate request directly with the Applicant Representative or with an authoritative source within the Applicant's organization, such as the Applicant's main business offices,  human resource offices, information technology offices, or other division that the CA deems appropriate.
> 
> In addition, the CA SHALL establish a process that allows an Applicant to specify the individuals who may request Certificates. If an Applicant specifies, in writing, the individuals who may request a Certificate, then the CA SHALL NOT accept any certificate requests that are outside this specification. The CA SHALL provide an Applicant with a list of its authorized certificate requesters upon the Applicant's verified written request.

@techliaison @LarryFrank

### LINKED ISSUE #171: control definition
Definitions 2 & 3 of Control  (lines 190-192) seem out of place for a Government document

### LINKED ISSUE #173: authentication for revocation request
Rather than "no stipulation" for identification and authentication for revocation requests I would think the request for revocation to be honored should either come from an authorized representative of the subscriber's organization or somehow be associated with the cert - like using the private key - otherwise it must be authenticated as official, unless the request is due to suspected compromise of the key.  
at a minimum I would expect a reference to 4.9.

### LINKED ISSUE #174: 4.5.2
this section header is about cert usage not cert validation - the text about RPs process & comply with the revocation info belongs in 4.9.6

should instead mention that RPs should comply with the KU & EKU included in the cert to match the header

### LINKED ISSUE #186: CA licensing
since the US has no rules for licensing CAs #4 should this be left off of the list in section 8

### LINKED ISSUE #193: Inconsistent requirements for subject name
There is inconsistent information in the certificate policy and profile about the contents of the subject field in domain validated server authentication certificates.

The certificate profile says that the subject field contains "cn=<one domain name>," however, Section 3.1.3 of the CP says that the subject field also contains a countryName:
```
Subscribers are not identified in Domain Validation certificates. Only the country (US) and domain name is included in the subject information.
```
Meanwhile, Section 7.1.4.2.2 contains generic rules of "optional," "required if," "prohibited if," etc. If the  profile is correct, then it seems that Section 7.1.4.2.2 should just say that CN is mandatory for all certificates; "S=District of Columbia," "O=U.S. Government," and "c=US" are mandatory for OV certificates but prohibited for DV certificates; and all other attributes are prohibited.

Does the subject name need to be exactly as described in the profile or is some flexibility permitted?

Field or Extension | Domain Validation | Organization Validation
-- | -- | --
Subject Identity Information | cn=\<one domain name\> | cn=\<one domain name\>,S=District of Columbia,O=U.S.Government,c=US

### LINKED ISSUE #197: How much to copy text from Baseline Requirements
Some of the text in the CP is more complicated than it needs to be since the text was copied from the Baseline Requirements document rather than being tailored to this particular CP. This was done in order to make it easier to demonstrate that the CP meets the Baseline Requirements, but it may also make it more difficult CAs to demonstrate that their CPSs satisfy the CP. I looked at some of the CPs listed at https://wiki.mozilla.org/CA/Included_Certificates and the ones I saw did not seem to copy language from Baseline Requirements in the same way this CP does.

I noticed this issue particularly in Section 3.2.2. Section 3.2.2.3, rather than copying the language from the Baseline Requirements seems to try to explain how this CP satisfies the requirements of Section 3.2.2.3 of the Baseline Requirements. Other sections, however, are more difficult to read.

Section 3.2.2.1, for example, basically copies the text from the Baseline Requirements, but then includes a practice note, which seems to negate the text before it.

I would like to propose that rather than copying the Baseline Requirements text here, the CP should simply say that the CA's responsibility is to verify that the applicant is part of the U.S. Federal government, and that doing so is sufficient to determine that attributes such as "c=US," "o=U.S. Government," and "st=District of Columbia" are valid for the certificate.

### LINKED ISSUE #312: Section General
**Organization / Program:** DoD <br>**Section:** General <br>**PDF Page:** General<br>**PDF Line(s):** None<br>**Comment:** It is difficult to determine what overall end-to-end process for certificate request and issuance this CP is intended to support. Individual requirements are listed, but they are very disjoint<br><br>**Suggested Change:** Identify one or more end-to-end process use cases for certificate request, approval, and issuance and then update section 3 and 4 to reflect the minimum requirements for these use cases to provide continuity in the document.

Also consider updating section 1.3 to provide details on the roles and responsibilities for individuals, organizations, and the system or device subscribers themselves in the process

### LINKED ISSUE #314: Section General
**Organization / Program:** DoD <br>**Section:** General <br>**PDF Page:** General<br>**PDF Line(s):** None<br>**Comment:** The intended audience for this CP is federal agencies, and the architecture described in the PowerPoint deck shows a limited number of issuing CAs, which would result in some agencies operating CAs that issue certificates to other agency subscribers. However, the CP doesn't clearly indicate what existing processes and roles could be leveraged and what would need to be different to issue certificates under this CP.<br><br>**Suggested Change:** Either use consistent terminology or provide clear mapping between terms used in other FPKI CPs so agencies understand how to use this capability.

Also, where possible, mirror existing process requirements to support agency deployments

### LINKED ISSUE #318: Section 1.1
**Organization / Program:** DoD <br>**Section:** 1.1 <br>**PDF Page:** 10<br>**PDF Line(s):** 27-28<br>**Comment:** None<br><br>**Suggested Change:** Include the certificate profiles in the document and in Table of contents

### LINKED ISSUE #323: Section 1.3.2
**Organization / Program:** DoD <br>**Section:** 1.3.2 <br>**PDF Page:** 12<br>**PDF Line(s):** 50-76<br>**Comment:** This section is supposed to define registration authorities for this PKI. The first paragraph seems to be completely unrelated to the topic, and the remaining text has a bunch of requirements that aren't necessarily related to registration authorities. Not sure why all that text is here, instead of somewhere in sections 3 or 4 where it talks about the process of requesting and issuing certificates. Furthermore, in Section 5 the term "Officer" is used, but says that officers act as registration authorities and validation specialists. That language should appear here to orient the reader.<br><br>**Suggested Change:** Move unrelated text to an appropriate section. Address the relationship between Ras, officers, and validation specialists here.

### LINKED ISSUE #336: Section 2: Add CT to publishing info
**Organization / Program:** DoD <br>**Section:** 2 <br>**PDF Page:** 24<br>**PDF Line(s):** N/A<br>**Comment:** There is a requirement later in the document to publish information to certificate transparency repositories, this requirement should be listed in section 2<br><br>**Suggested Change:** Add information about publication requirements for certificate transparency to section 2.

### LINKED ISSUE #338: Section 3.1.3
**Organization / Program:** NSA <br>**Section:** 3.1.3 <br>**PDF Page:** 26<br>**PDF Line(s):** 460<br>**Comment:** Pick a definition for 'Subscriber' and stick to it.  Either it is the device, or it is the person behind the device.  In this section, it appears to be the person behind the device.  For most of the rest of the document, it appears to be the device.  This section should state that the CA doesn't issue anonymous certificates.<br><br>**Suggested Change:** Replace with, "A CA SHALL NOT issue anonymous certificates.  CA certificates SHALL NOT contain anonymous or pseudonymous identities."

### LINKED ISSUE #415: Section 6.1.6
**Organization / Program:** NSA <br>**Section:** 6.1.6 <br>**PDF Page:** 62<br>**PDF Line(s):** 1585-1587<br>**Comment:** Why are two methods listed?  Choose the most robust and use that.<br><br>**Suggested Change:** None

### LINKED ISSUE #416: Section 6.1.7: Key Usage
**Organization / Program:** DoD <br>**Section:** 6.1.7 <br>**PDF Page:** 62<br>**PDF Line(s):** 1588<br>**Comment:** Text in this section addresses Root CA private keys, but does not address key usage purposes for all certificates.<br><br>**Suggested Change:** Add requirements for subscribers

### LINKED ISSUE #434: Section 7.1.2.2
**Organization / Program:** DoD <br>**Section:** 7.1.2.2 <br>**PDF Page:** 70<br>**PDF Line(s):** 1837-1838<br>**Comment:** See comment on the profiles.  Why can this not be a single certificate since in most cases it will be a single certificate.<br><br>**Suggested Change:** None

### LINKED ISSUE #441: Section 7.1.6
**Organization / Program:** DoD <br>**Section:** 7.1.6 <br>**PDF Page:** 76<br>**PDF Line(s):** N/A<br>**Comment:** None<br><br>**Suggested Change:** Reconcile test certificate discussion and other comments and determine if Test Certificate CABF OID (2.23.140.2.1) is also required

### LINKED ISSUE #442: Section 7.1.9
**Organization / Program:** DoD <br>**Section:** 7.1.9 <br>**PDF Page:** 77<br>**PDF Line(s):** 2100<br>**Comment:** This statement contradicts Test Certificate requirement (ambiguity there as noted in a comment) and lines 1866 and 1824<br><br>**Suggested Change:** Reconcile

### LINKED ISSUE #450: Section 8.4
**Organization / Program:** DoD <br>**Section:** 8.4 <br>**PDF Page:** None<br>**PDF Line(s):** None<br>**Comment:** Allow for government audit - allowed under the BD and what is done in DoD.  Current Common/DoD CP requires that the auditor check for conformance with all aspects of the CP and the CPS.  "verify that the audited party has in place, a system to assure the quality of the services that it provides, and that it complies with all of the requirements of this CP and its CPS. All aspects of the audited party’s operation related to this CP shall be subject to compliance audit inspections."<br><br>**Suggested Change:** Add third bullet "3. A government directed audit scheme that consists of comparable criteria to a or b above and is available for public review."

### LINKED ISSUE #452: Section 8.5
**Organization / Program:** DoD <br>**Section:** 8.5 <br>**PDF Page:** 81<br>**PDF Line(s):** 2204-2208<br>**Comment:** Section references the auditor finding a discrepancy between the requirements of the CP or CPS and the design, operation, or maintenance of the CAs. However, a WebTrust audit doesn't necessarily audit against the CP and CPS, it audits against the WebTrust guidelines, so this requirement doesn't make sense<br><br>**Suggested Change:** Change "Certificate" to "Certification"

### LINKED ISSUE #464: Section 1 Editorial
Additional comments received on Section 1 

| Organization / Program | PDF Line | Comment | Change |
| --- | --- | --- | --- |
| FPKIMA|4|Suggest adding "in the" after USFPKI. Confusing the way it is currently worded.|U.S. Federal Public Key Infrastructure in the issuance and management of publicly trusted device certificates." |
| FPKIMA|16|Take out the "be"|The requirements are designed to flow down from the Root…. |
| FPKIMA|18|Harder to remember to update the BR version when its hidden in the paragraph.|Introduce a call-out that of all the public trust requirements that are met. CAB BRs, Microsoft v, Mozilla v… etc. |
| FPKIMA|193-194|The definition of a country may not always align with the U.S. Government position.  Other nations may recognize another country that the U.S. Government may not. |Since these CAs will only issue to the U.S. Government, remove the definition for country or Country: A geographic region as recognized by the U.S. Government. |
| FPKIMA|219|~~The definition of Effective Date has an actual date.~~|~~Effective Date: "The date upon which something is considered to take effect, which may be a past, present or future date. This may be different from the date upon which the event occurs or is recorded."  https://en.wikipedia.org/wiki/Effective_date~~|
| FPKIMA|234-236|The High Security Zone definition includes the text "or Delegated Third Party"  However, the CP states previously that Delegated Third Parties are not authorized.  This text should be removed.|High Security Zone: An area (physical or logical) protected by physical and logical controls that appropriately protect the confidentiality, integrity, and availablity of the CA's private key or cryptographic hardware. |
| FPKIMA|311|Using Requirements to define Requirements is circular logic.|Requirements: a thing that is compulsory; a necessary condition. |
| FPKIMA|39|PKI Authorities include more than the FPKIPA.  This section, at the very least, needs to include  paragraphs for Federal CIO Council, FPKIPA, and FPKIMA.|Use the X.509 Certificate Policy for the US Federal PKI Common Policy Framework, paragraph 1.3, "PKI Participants" as the model for this section re-write. |
| FPKIMA|39|"Certificate Status Servers" needs to be included as a paragraph in the "PKI Authorities" section.|Use the X.509 Certificate Policy for the US Federal PKI Common Policy Framework, paragraph 1.3.1.7, "Certificate Status Servers" as the model to add to this section. |
| FPKIMA|79|The Definition Section in 1.6.1 should define WHAT is a subscriber.  This section should define WHO are the subscribers.|Use the X.509 Certificate Policy for the US Federal PKI Common Policy Framework, paragraph 1.3.4, "Subscribers" as the model for this section re-write. |
| FPKIMA|81|The Definition Section in 1.6.1 should define WHAT is a Relying Party.  This section should define WHO are the Relying Parties.|Use the X.509 Certificate Policy for the US Federal PKI Common Policy Framework, paragraph 1.3.5, "Relying Parties" as the model for this section re-write. |
| FPKIMA|86|"Compliance Auditors" should be named "Other Participants"|Use the X.509 Certificate Policy for the US Federal PKI Common Policy Framework, paragraph 1.3.6, "Other Participants" as the model for this section re-write. |
| FPKIMA|88 & 92|Alignment and consistency with Public Trust Feedback -- explicitly state SSL/TLS, Certificate Transparency, etc.|Align these sections to ensure consistency with Slide 25, "What is in Scope?" of PublicTrust_PKI_RequestFeedback_2017.pdf |
| FPKIMA|234|Consider renaming "High Security Zone" to avoid confusion between the FISMA definition of High Security Impact systems. |Rename this term, "Enhanced Security Zone", and a footnote that CAB Forum uses different terminology which has FISMA connotations. |
| FPKIMA|362|Convert from table format to be consistent with other sections|Remove table |
| FPKIMA|no line number|Within the Acronym table, "CAA Certification Authority Authorization" should be defined as well|Include a definition in Section 1.6.1 for CAA Certification Authority Authorization" |
| FPKIMA|no line number|Missing acronyms|Include acronyms for: HTTP, HTTPS, SCT, CT |
| DoD|29-30| |Add "United States (US)" before the word "Federal" |
| DoD|Table before line 32| |Replace the term "fingerprint" with hash |
| DoD|Table| |Put place holder n and m at the end of the two OIDs to make OIDs distinct.  Or put TBDs at the end |
| DoD|Table| |Do not bold the OID prefix.  It gives the impression that is the actual OID |
| DoD|Table| |Restructure so the OID prefix (the CSOR arc) is only rendered once |
| DoD|Table| |Replace TBD for Change Proposal to None since version 1.0 will be the original CP. |
| DoD|Table| |Change Effective column with Effective Date and delete * and associated text.  Note there is a separate section for compliance |
| DoD|77|This sentence is not clear.  What is delegated third party and what does it mean to not allow them.|Delete the sentence or add more meat around it so that it is clear. |
| DoD|94-98|Note that even when a TLS Server acts as a TLS client, it is because it is being accessed as a TLS Server.|It would be better to explicitly and directly state that certificates issued by the TLS Server PKI can only be used during the course of TLS Server Authentication. |
| DoD|111| |Replace "has inclusion in" with "is included in" |
| DoD|115|Text is not a complete sentence.|Replace text with a complete sentence - e.g., "The FPKIPA shall affirm the suitability of any CPS to this policy." |
| DoD|124-397|The glossary section is very long, and it is hard to find the acronyms table when reading through the policy, both of these problems impact overall readability|Move the glossary and acronyms to an appendix at the end of the document |
| DoD|128| |Fix spelling of "physically" |
| DoD|128-129|The term air-gapped is used in only three places and two of them are in definition.  To be consistent and to be unambiguous define it to be disconnected from any network|Change the definition to "Certificate Systems or components that are physically and logically disconnected from other networks" |
| DoD|131| |Change "issues" to "is issued" |
| DoD|147-148| |Change "an entity's" to "the audited entity's" |
| DoD|205| |Change "contract" to "contact" |  
| DoD|253| |Delete "alphanumeric" since OID are hierarchy of numeric numbers.  There is no alpha in OID.  The numerics are given names at each level for ease of use. |
| DoD|259-260|Online should include the concept of being "powered on"|Add "powered on" |
| DoD|261-262| |Change the definition per RFC 6960 to "A protocol to determine revocation status of one or more certificates without the use of CRLs.  The protocol is defined in RFC 6960" |
| DoD|274-275|There is no such thing as "corresponding root certificate"|Change the definition to "A Certificate that is trusted by virtue of the fact that a certification path exists to it from a Root Certificate distributed as a trust anchor in widely-available application software." |
| DoD|315-316|While the CP does not use the term, the certificate profiles use the term intermediate CA|Revise the definition of Root CA to say "Intermediate CA and Subordinate CA certificates" |
| DoD|346-348|It is not clear if this OID is supposed to appear in certificate policies extension or is a new extension.  Research indicates that it may appear in certificate policies extension.|Clarify where this OID appears or if this is an OID for an extension.  If this is an extension, document its syntax.  Include the actual OID in the CP which appear to be 2.23.140.2.1 |
| Army CIO/G-6|349|The CP does not have a definition for Transport Layer Security (TLS) |Recommend adding a definition for Transport Layer Security (TLS) to this section. Recommend adding parts of RFC 6962, Page 13, Section 3.3 to the document. This section references the inclusion of a signed certificate timestamp in the TLS handshake. |
| DoD|356-357|Validity Period is a specific term in X.509.  Note that a certificate can be back dated or its validity can start sometime in the future.  I have revised it and added something about not before that may or may not be true. |Revise the definition to state Validity Period consists of two dates: date when the certificate becomes valid (called not before) and date when the certificate expires (not after).  This CP requires the CAs to assert not before date to be the date certificate is issues and not after date to be some time after that. |
| SSA|Line 124|The term "natural person" is used throughout the document; however, it is not defined. A definition of the term will improve readers' understanding.|We suggest including a definition for the term "natural person" in Section 1.6.1 |
| SSA|2|In the Introduction section, Overview subsection, the document describes background and scope in a general narrative. The document would have better readability if the section was separated into background and scope subsections.|We suggest inserting subsections for Background and Scope and separating the language appropriately. |
| DHA| |The title of the CP "Federal Public Trust Device Certificate Policy" is described as the "Federal Public Device PKI Certificate Policy"  not a big deal.| |
| DHA| |Since the CP cites RFC 6844, it should be added to 1.6.3 with the other RFCs| |
| DHA| |~~The effective date in 1.6.1 looks like the author just copied the section from the draft CA / Browser Forum Baseline Requirements Version 1.5.2.~~ | |
| DHA| |The CP cites RFC 5280, does it need anything from RFC 6818| |
| Slandala|219|~~Effective Date?~~| |

### LINKED ISSUE #472: General Editorial (No Section Specified)
Additional comments received and labeled by submitter as "General"

| Organization / Program |  Comment | Change |
| --- |  --- | --- |
| FPKIMA |  Throughout the document, i.e. and e.g. are used incorrectly. i.e. means id est, or "that is"e.g. means exempli gratia, or "for the sake of example" | Review usage of i.e. and e.g. For example, line 258 should be e.g., and line 614 should be i.e. |
| FPKIMA |  Usage of FPKIPA / Policy Authority / FPKI Policy Authority is inconsistent.  |  FPKIPA or Policy Authority should be chosen, and used throughout the document. |
| FPKIMA |  SHALL' and 'shall' are used interchangably throughout the document.  The casing of the word should be consistent througout. | Change all 'SHALL' instances to 'shall'. |
| FPKIMA |  There are a lot of references to variations of "Version 1.4.9 of the guidelines adopted by the Certification Authority/Browser Forum (“CA/Browser Forum”)".   However, this Root Certificate should be in multiple trust stores. |At a minimum, remove the version number specified, however, since the requirements have already been incorporated into the CP, the entire reference could be removed. |
| FPKIMA | The CP uses inconsistent wording when referencing other CAs that fall under this CP.  For example, it uses "the CA" vs "all CAs" vs "each CA". | Change to "issuing CAs" where appropriate, or "the CA." |
| DoD | Format - is there any reason the level 1 and level 2 headers have such a large font size? While minor, it does impact readability. | Reduce the font size for level 1 and level 2 headers |
| DoD | Terminology - CA is defined as an organization in the glossary, but it is also used to define the system that issues certificates. Need to address this discrepancy. | Either include the systems in the definition of CA, or use different terms when referring to the responsible organization or the system that issues certificates. |
| DoD | Acronym usage - many terms, such as CP, CPS, and FPKIPA are defined with acronyms but then throughout the document the acronyms are not used consistently. Sometimes the acronym is used and sometimes the terms are spelled out | If going to define an acronym, use it consistently throughout the document |
| DoD | Terminology - while having the glossary is helpful for reference, terms that are not commonly used should also be defined where they are first used within the body of the document rather than forcing the reader to turn back to the glossary to figure out what the section is talking about | Add information regarding the meaning of terms to the document body when they are first used |
| DoD |Format - There is no reason to capitalize the words "shall," "should," "may," etc. Stating how they are used in the document and then using them consistently is sufficient. Capitalizing them every time they are used makes the document very difficult to read (and currently, the document is not internally consistent)  | PLEASE do not capitalize these terms in the document |
| DoD | Terminology - the terms "shall" and "must" are synonyms according to RFC 2119. This document sometimes uses "shall" and sometimes uses "must" with no apparent reason behind it, and no stated description of why one or the other is used. Also, there are times when "are" or "will" are used instead of either "shall" or "must" (or in a couple of places "not allowed to" instead of "shall not"). This lack of consistency is very confusing to the reader  | Either pick one and use it exclusively (recommend "shall") or provide a description of what it means when one is used or the other one is used and stick to that convention. Document needs a complete review for the use of appropriate policy language. |
| DoD | CPS references - the purpose of a CPS is to describe how a CA will implement the requirements identified in a certificate policy. As a result, the certificate policy does not need to say "…shall describe…in the CPS." There are a number of sections in this certificate policy where that language appears (see sections 3.2.1, 3.2.2, 3.2.2.3, 4.1.2, 4.9.3, 5.5.7, 6.1.1.1) | Delete all statements that a CPS will describe something, and replace with what the minimum requirement is. |
| DoD | Cross references - dropping numbers into the document without providing context for what they are for is confusing to the reader. | Recommend saying "Section..."  or even "the requirements in section…" before listing cross referenced sections.. |
| DoD | Terminology - references to the FPKIPA are inconsistent. Sometimes it is referred to as just the policy authority, other times the acronym FPKIPA is used | Need to be consistent so the reader knows what entity is being referred to. Recommend using FPKIPA consistently throughout the document|
| DoD | List capitalization - sometimes bulleted and numbered lists all start with a capital letter and sometimes they don't | Pick a format and use it consistently throughout the document |
| DoD | Calling this "device PKI" is a misnomer since this is only for TLS Servers | The title as well as throughout the document, change "device PKI" to "Web PKI" or "TLS Server" PKI |
| DoD |  | Replace the term Certificate Authority with Certification Authority or CA.  Lines 58-61 on page 12 in Section 1.3.2 are an example. |
| DoD | Replace "MUST" with "SHALL" throughout per previous discussion. |  |
| DoD | Replace the term Auditor with Compliance Auditor to disambiguate overloaded term and in few places replace Auditor with Audit Administrator as applicable |  |

### LINKED ISSUE #481: CA terminology: Fix
Choose and fix throughout the CP the use of: 

- The CA
- All CAs
- Issuing CAs (which is all CAs - but used in the possessive in some areas; "the Issuing CA" is the issuer of the certificate and requirement impacting that certificate)

### LINKED ISSUE #497: RFC 6962 - CT References
- [x] References for CT need to be checked
- [x] Pre-certificate definition changes between RFC 6962 and RFC 6962-bis

### PR DESCRIPTION
**Pull request by lachellel**
There are a lot of changes in this PR related to formatting and the site prep.

A number of final open issues were also reviewed and updates made as needed.  

closes #497 closes #481 closes #472 closes #464 closes #452 closes #450 closes #442 closes #441 closes #434 closes #416 closes #415 closes #338 closes #336 closes #323 closes #318 closes #314 closes #312 closes #197 closes #193 closes #186 closes #174 closes #173 closes #171 closes #154

### CODE REVIEW

**Comment by lachellel** [comment]
@techliaison - this PR should have your edits **and** the website versions should be displaying 

https://federalist-proxy.app.cloud.gov/preview/uspki/policies/topdown/

File: certificate-policy.md [hunk 1]
```diff
@@ -971,44 +972,44 @@ CA archive records shall be sufficiently detailed to determine the proper operat
...
 
-## 5.7 Compromise and disaster recovery
+### 5.7 Compromise and disaster recovery
 
-### 5.7.1 Incident and compromise handling procedures
+#### 5.7.1 Incident and compromise handling procedures
 CAs shall have an Incident Response Plan and a Disaster Recovery Plan.  The CA is not required to publicly disclose the Incident Response Plan and Disaster Recovery Plan but shall make the plans available to the CA's Qualified Auditor upon request.
 
-The FPKIPA shall be notified by the CAs operating under this policy of any incident. An incident is defined as a violation or imminent threat of violation of this CP, the CA's CPS, Memorandums of Agreements, or any other document that governs the operations of the CA. An incident may include but is not limited to the following:
+The FPKIPA shall be notified by the CAs operating under this policy of any incident. An incident is defined as a violation or imminent threat of violation of this CP, the CA's CPS, government memorandum of agreements, or any other document that governs the operations of the CA. An incident may include but is not limited to the following:
```
Comment (konklone) [inline certificate-policy.md]: Nit: I think `Memoranda of Agreement` is probably the right pluralization. :)

File: certificate-policy.md [hunk 2]
```diff
@@ -1770,8 +1768,7 @@ No stipulation.
 
 **Base Domain Name**: The portion of an applied-for FQDN that is the first domain name node left of a registry-controlled or public suffix plus the registry-controlled or public suffix (e.g. "example.co.uk" or "example.com"). For FQDNs where the right-most domain name node is a gTLD having ICANN Specification 13 in its registry agreement, the gTLD itself may be used as the Base Domain Name.
 
-**Baseline Requirements**: The Baseline Requirements for the Issuance and Management of Publicly-Trusted Certificates as
-published by the CAB Forum (http://www.cabforum.org).
+**Baseline Requirements**: The Baseline Requirements for the Issuance and Management of Publicly-Trusted Certificates as published by the CAB Forum (http://www.cabforum.org).
```
Comment (konklone) [inline certificate-policy.md]: Nit: the CABForum website is now `https://cabforum.org`. (I'm sure this is also outdated in the BRs, and the domain is preloaded, but would be nice to use `https://` URIs where we can.)

File: certificate-policy.md [hunk 3]
```diff
@@ -1884,22 +1879,12 @@ published by the CAB Forum (http://www.cabforum.org).
...
 
 **Required Website Content**: Either a Random Value or a Request Token, together with additional information that uniquely identifies the Subscriber, as specified by the CA.
 
 **Reserved IP Address**: An IPv4 or IPv6 address that the IANA has marked as reserved:
-
-[http://www.iana.org/assignments/ipv4-address-space/ipv4-address-space.xml](http://www.iana.org/assignments/ipv4-address-space/ipv4-address-space.xml)
-
-[http://www.iana.org/assignments/ipv6-address-space/ipv6-address-space.xml](http://www.iana.org/assignments/ipv6-address-space/ipv6-address-space.xml)
+- [http://www.iana.org/assignments/ipv4-address-space/ipv4-address-space.xml](http://www.iana.org/assignments/ipv4-address-space/ipv4-address-space.xml)
+- [http://www.iana.org/assignments/ipv6-address-space/ipv6-address-space.xml](http://www.iana.org/assignments/ipv6-address-space/ipv6-address-space.xml)
```
Comment (konklone) [inline certificate-policy.md]: Nit: all these iana.org URLs can be accessed at `https://` URIs.

File: pages/00-background.md [hunk 4]
```diff
@@ -5,13 +5,11 @@ title: Background
 permalink: /
 ---
 
-Securing Federal Government web services, and delivering information to our consumers and partners in a trusted and privacy-enhancing manner, is a priority for Federal Government teams across all our missions.
+Securing Federal Government web services and delivering information to our consumers and partners in a trusted and privacy-enhancing manner is a priority for Federal Government teams across all our missions.
 
- - Office of Management and Budget M-15-13, [“A Policy to Require Secure Connections across Federal Websites and Web Services”](https://https.cio.gov/){:target="_blank"}
+Federal Government teams are working to ensure our missions have secure, cost-effective, and stable _services_ to secure these web services.   We developed this [Certificate Policy]({{site.baseurl}}/certificatepolicy) to address Internet PKI requirements and to support the establishment of the **new** U.S. Federal Public Trust TLS Public Key Infrastructure (PKI) for .mil and .gov websites.
```
Comment (konklone) [inline pages/00-background.md]: Suggestion: `websites` -> `web services`. We're already using `web services` elsewhere in this page, and it sounds more serious (and accurate) to emphasize that this covers a variety of important and sophisticated services that are offered over HTTP, not just informational sites.

File: pages/00-background.md [hunk 5]
```diff
@@ -23,29 +21,29 @@ The browser community, commercial and non-profit trust store owners, and Standar
...
 - [Apple Trust Store Program](https://www.apple.com/certificateauthority/ca_program.html){:target="_blank"}
 - [Mozilla Community and Trust Store Program](https://www.mozilla.org/en-US/about/governance/policies/security-group/certs/){:target="_blank"}
 - [Google Chromium](https://www.chromium.org/Home/chromium-security/certificate-transparency){:target="_blank"}
-- Federal Government security requirements, standards and missions
 
 This Certificate Policy was scoped to encompass:
 
 - _Organizational Validation_ and _Domain Validation_ certificates for domains
-- .mil and .gov web services intended to be trusted by the public
+- .mil and .gov web services intended to be trusted by consumers and partners on the public Internet
```
Comment (konklone) [inline pages/00-background.md]: Happy to dicker over this some other time/place, but I think it's more important to emphasize the external nature of the users/devices, than to emphasize the internet as the network which will be serving these things. In addition to some inevitable and appropriate use of these certificates for "intranet" services, I think "consumers and partners" might trivialize the size/scope of the audience.

Suggestion:

> ...trusted by public users, consumers, and partners

**Review by konklone — APPROVED** [review]
I left some nits, and a suggested text change. No showstoppers. :+1:

File: certificate-policy.md (hunk 2 above)
Comment (lachellel) [reply certificate-policy.md]: Oh funny - good catch.  I had changed it in Section 1 to be https (for the same reason - i dislike having http in gov docs!).  I should have done a grep/search throughout.

**Comment by lachellel** [comment]
I added the https references. 

Two minor nits: 

@dcooper16 - when is NIST going to switch all to https?  :+1:   I couldn't update references to nist docs to use https (the docs are behind http only - nudge nudge wink wink)

www.webtrust.org - @konklone - seriously? webtrust itself as an org doesn't have https?

**Comment by dcooper16** [comment]
I only found two references to NIST websites: links for SP 800-89 and SP 800-56Ar2. I can't say when the server that hosts those files will be configured to support HTTPS, but you could use the following links for those documents: https://doi.org/10.6028/NIST.SP.800-89 and https://doi.org/10.6028/NIST.SP.800-56Ar2.

At NIST, we've been encouraged to use the [DOI](https://csrc.nist.gov/Publications/DOI) URL whenever possible. If you use the DOI URLs, then you won't have to update the URLs in your document even if NIST reorganizes its website.

There are also DOI URLs available for every IETF RFC, e.g., https://doi.org/10.17487/RFC6844.

**Comment by lachellel** [comment]
thanks!  To be honest, I haven't quite understood why the doi.org URLs are sent out and also listed.  It does make a lot more sense. 

[We have a lot of links to NIST docs on our websites and in documents...a LOT... @djpackham - see David Cooper's note.  At some point we should update all our NIST references to use the doi.org URIs]

**Comment by konklone** [comment]
> At NIST, we've been encouraged to use the DOI URL whenever possible. If you use the DOI URLs, then you won't have to update the URLs in your document even if NIST reorganizes its website.

@dcooper16 I get the idea, but that strikes me as short-sighted in a different way. It's nice to be able to have URLs keep working even if NIST reorganizes, but it's nicer to have URLs keep working even if the [International DOI Foundation](https://www.doi.org) goes away. 

NIST should use its own proper permalink system, redirecting as necessary, and maintain those redirects as the underlying site architecture changes. That way, links to NIST resources use nist.gov domains.

The Government Publishing Office does exactly this, providing a link service to official government documents: https://github.com/usgpo/link-service (And this will be especially helpful as they reorganize, and transition from the FDSys system to GovInfo over 2018.)

**Pull request merged**